test/corpus/rule.mk

```diff
@@ -269,38 +269,6 @@ target:
       (recipe_line
         (shell_text)))))
 
-================================================================================
-Rule, recipe, single line, custom .RECIPEPREFIX I (TODO)
-================================================================================
-.RECIPEPREFIX = >
-
-target:
->echo "foobar"
-
---------------------------------------------------------------------------------
-
-
-================================================================================
-Rule, recipe, single line, custom .RECIPEPREFIX II (TODO)
-================================================================================
-.RECIPEPREFIX = a
-
-target:
-aecho "foobar"
-
---------------------------------------------------------------------------------
-
-
-================================================================================
-Rule, recipe, single line, custom .RECIPEPREFIX III (TODO)
-================================================================================
-.RECIPEPREFIX = >
-
-target: ;
->echo "foobar"
-
---------------------------------------------------------------------------------
-
```

Comment:
I removed these since they are known-failing.  This isn't integral to my changes.